doc/components/floating_point.md

```diff
@@ -6,13 +6,16 @@ Floating-point operations require meticulous precision, and have standards like
 
 The `FloatingPointValue` class comprises the sign, exponent, and mantissa `LogicValue`s that represent a floating-point number. `FloatingPointValue`s can be converted to and from Dart native `Double`s, as well as constructed from integer and string representations of their fields.  They can be operated on (+, -, *, /) and compared.
 
-The various IEEE constants representing corner cases of the field of floating-point values for a given size of `FloatingPointValue`: infinities, zeros, limits for normal (e.g. mantissa in the range of [1,2]) and sub-normal numbers (zero exponent, and mantissa <1).
+A `FloatingPointValue` has a mantissa in $[0,2)$ with $$0 <= exponent <= `maxExponent`$$ A normal `isNormal` `FloatingPointValue`
```

Comment:
what are these `$`'s about?  looking at it in markdown doesn't fully clarify
https://github.com/intel/rohd-hcl/blob/edc71c6569724e99fa8aa9c40968ef75bc95df77/doc/components/floating_point.md

Reply:
I looked up github markdown for equations.   Inline is bracketing by a single Dollar sign.  Equation is using a double Dollar sign before and after. I probably need to remove the quotes as they do not work in equations.
![image](https://github.com/user-attachments/assets/a9c1e350-94fb-44db-948e-fc13bba28395)

Reply:
oh wow that looks really cool on yours!  why does it look like this to me in github?
![image](https://github.com/user-attachments/assets/deb09603-be01-4a07-954f-5af7e2404f9c)

perhaps there's a way to make this work nicely in github's markdown parsing as well?

Reply:
Let me figure out why this is.  Github is supposed to support this format feature.

Reply:
https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions

Reply:
This appears correctly on github:
![image](https://github.com/user-attachments/assets/646c2221-ac89-423b-b357-aa76357e2bd7)

Reply:
indeed it does! nice!